src/bsp/platform/cc3200/xi_bsp_fwu_cc3200.c

```diff
@@ -76,7 +76,10 @@ void xi_bsp_fwu_order_resource_downloads( const char* const* resource_names,
 
 void xi_bsp_fwu_on_package_download_finished( const char* const firmware_resource_name )
 {
-    ( void )firmware_resource_name;
+    if ( NULL == firmware_resource_name )
```

Comment:
JC, sorry for nitpicking here, but I would do the opposite:
if ( NULL != firmware_resource_name)
{
include here the cc3200 firmwareupdate / reboot specific code
}

I think without any comment this code above tells more the intent: if there is a firmware_resource_name partition switch and reboot is necessary.
The current version tells me this: if there is no firmware_resource_name then there is nothing to do (even if this is true now it not necessary the case, one may do something with the downloaded package here).

And yes, nitpicking, sorry.

Reply:
Sure, no problem, change implemented :)